### PR TITLE
Stream line code of consumer model

### LINKED ISSUE #152: StreamLine code of Consumer Module

### PR DESCRIPTION
**Pull request by ankit213**
Fixed #152 

Changed Files:
1.**ConsumerAppRepositoryTest.cs**: update naming conventions 
2.**ConsumerAppController.cs**: update api route path
3.**ConsumerAppNotFound.cs**:add customer exception class
4.**IConsumerAppRepository.cs**:update naming conventions
5.**consumerapp.service.ts**:update services rateled api route path

### CODE REVIEW

File: Promact.Oauth.Server/src/Promact.Oauth.Server/Controllers/ConsumerAppController.cs [hunk 1]
```diff
@@ -148,33 +150,28 @@ public ConsumerAppController(IConsumerAppRepository iConsumerAppRepository, User
...
+                consumerApp.CallbackUrl = consumerAppsAc.CallbackUrl;
+                consumerApp.Description = consumerAppsAc.Description;
+                consumerApp.UpdatedDateTime = DateTime.Now;
+                if (await _consumerAppRepository.UpdateConsumerApps(consumerApp) != 0)
+                    return Ok(true);
+                else
+                    return Ok(false);
             }
-            catch (Exception ex)
+            catch (Exception)
```
Comment (chintans) [inline Promact.Oauth.Server/src/Promact.Oauth.Server/Controllers/ConsumerAppController.cs]: This is a generic exception. When will this exception be thrown?

File: Promact.Oauth.Server/src/Promact.Oauth.Server/Repository/ConsumerAppRepository/ConsumerAppRepository.cs [hunk 2]
```diff
@@ -52,34 +53,39 @@ public ConsumerAppRepository(IDataRepository<ConsumerApps> appsDataRepository, I
...
+                consumerAppObject.AuthId = GetRandomNumber(true);
+                consumerAppObject.AuthSecret = GetRandomNumber(false);
                 consumerAppObject.CreatedDateTime = DateTime.Now;
                 _appsDataRepository.AddAsync(consumerAppObject);
                 await _appsDataRepository.SaveChangesAsync();
                 return consumerAppObject.Id;
             }
-            return 0;
+            else
+                return 0;
```
Comment (chintans) [inline Promact.Oauth.Server/src/Promact.Oauth.Server/Repository/ConsumerAppRepository/ConsumerAppRepository.cs]: Why returning 0?

File: Promact.Oauth.Server/src/Promact.Oauth.Server/Repository/ConsumerAppRepository/ConsumerAppRepository.cs [hunk 3]
```diff
@@ -52,34 +53,39 @@ public ConsumerAppRepository(IDataRepository<ConsumerApps> appsDataRepository, I
...
         /// <returns></returns>
-        public async Task<List<ConsumerApps>> GetListOfApps()
+        public async Task<List<ConsumerApps>> GetListOfConsumerApps()
         {
             return await _appsDataRepository.GetAll().ToListAsync();
         }
 
         /// <summary>
-        /// This method used fro get apps object by id. -An
+        /// This method used foo get consumer app object by id. -An
```
Comment (chintans) [inline Promact.Oauth.Server/src/Promact.Oauth.Server/Repository/ConsumerAppRepository/ConsumerAppRepository.cs]: foo?

File: Promact.Oauth.Server/src/Promact.Oauth.Server/Controllers/ConsumerAppController.cs (hunk 1 above)
Comment (ankit213) [inline Promact.Oauth.Server/src/Promact.Oauth.Server/Controllers/ConsumerAppController.cs]: I think sometimes unexpected database error may occur, at that time generic exception is thrown.

File: Promact.Oauth.Server/src/Promact.Oauth.Server/Repository/ConsumerAppRepository/ConsumerAppRepository.cs (hunk 3 above)
Comment (ankit213) [inline Promact.Oauth.Server/src/Promact.Oauth.Server/Repository/ConsumerAppRepository/ConsumerAppRepository.cs]: it's by mistake, i have changed it.

File: Promact.Oauth.Server/src/Promact.Oauth.Server/Controllers/ConsumerAppController.cs (hunk 1 above)
Comment (chintans) [inline Promact.Oauth.Server/src/Promact.Oauth.Server/Controllers/ConsumerAppController.cs]: There can be no database error which is unexpected. That is why there is point keep generic exception here. Please handle it better.

Comment (ankit213) [inline Promact.Oauth.Server/src/Promact.Oauth.Server/Controllers/ConsumerAppController.cs]: I have removed generic exception and added **FailedToFetchDataException** custom exception when consumer app list empty.

**Pull request merged**